otherlibs/stdlib_alpha/effect.ml

```diff
@@ -896,18 +1295,105 @@ module Make_rec (Ops : Operations_rec)
         * ('o, ('a, 'es) result, 'es) Continuation.t
         -> ('a, 'es) result
 
+  (* Here and below, [portable] versions of functions are defined before
+     non-[portable] ones to avoid shadowing the original functions. *)
+
+  module Portable = struct
+    let portable_fiber (type a b) f =
+    (* CR dkalinichenko: here and below, we use [magic_portable]
+       to indicate that the handler is [portable] when it receives
+       [contended] values.
+      
+       We may be able to rewrite this to avoid magic. *)
```

Comment:
Rather than have lots of repeated `Obj.magic_portable`s, could you instead use just one to create a toplevel `fiber_portable` function that is equal to the toplevel `fiber` function.

Reply:
Done, now everything lives in the `DRF` module.